PythonPonies/bangazon-storefront
# fixed

# Description
Add description about what this pull request covers.
Few tweaks for routes on the site
## Number of Fixes
Type of number of fixes included in this PR.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request

## Problem to Solve
Describe the problem(s) this pull request solves.
- fixed issue with payment view and buy product when not logged in
## Proposed Changes
Describe the proposed changes.

## Expected Behavior
Describe the expected behavior.

## Steps to Test Solution

1. git fetch
2. runserver
3. log out
4. go to product detail and see no button to buy product
5. log in and checkout, and don't get an error

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
